etna/analysis/plotters.py

```diff
@@ -678,34 +686,58 @@ def plot_correlation_matrix(
     ----------
     ts:
         TSDataset with timeseries data
+    columns:
+        Columns to use, if None use all columns
     segments:
         Segments to use
     method:
         Method of correlation:
 
         * pearson: standard correlation coefficient
-
         * kendall: Kendall Tau correlation coefficient
-
         * spearman: Spearman rank correlation
-
+    mode: 'macro' or 'per-segment'
+        aggregation mode
```

Comment:
Start the description with the capital letter
We actually don't have a convention about it, however all of them should start with the same type of the letter